src/common/libpmi/bizcard.h

```diff
@@ -20,6 +21,11 @@ void bizcard_decref (struct bizcard *bc);
 const char *bizcard_encode (const struct bizcard *bc);
 struct bizcard *bizcard_decode (const char *s, flux_error_t *error);
 
+/* N.B. bizcard_tojson() doesn't lend a reference (do not json_decref()!).
+ */
+struct bizcard *bizcard_fromjson (json_t *obj);
+json_t *bizcard_tojson (const struct bizcard *bc);
```

Comment:
Unlikely to be a real issue, but in most other places `_tojson()` functions return a `json_t *` with a reference, so this could be confusing. If the function didn't have the `to` preposition, it might be less likely to cause any confusion. Suggestion, rename function something like `bizcard_get_json()` (and perhaps return `const json_t *` as a reminder?)